aries-site/src/components/content/SubsectionText.js

```diff
@@ -20,11 +21,15 @@ const StyledBox = styled(Box)`
   }
 `;
 
-export const SubsectionText = ({ level, size, ...rest }) => (
-    <StyledBox width="large" margin={{ bottom: 'medium' }}>
-      <Paragraph size={size || TEXT_SIZE[level]} fill margin="none" {...rest} />
-    </StyledBox>
-  );
+export const SubsectionText = ({ children, level, size, ...rest }) => (
+  <StyledBox width="large" margin={{ bottom: 'medium' }}>
```

Comment:
one comment if you have all the styles above for the `const StyledBox = styled(Box)` why not put `marginBottom` in there?

Reply:
Good observation. I just added the `<HighlightPhrase />` portion and left the existing implementation alone.

Reply:
got it thank you for clarifying.